homeassistant/components/watchdog_file_watcher.py

```diff
@@ -0,0 +1,75 @@
+"""
+Component for monitoring activity on a folder.
+"""
+import asyncio
+import logging
+import voluptuous as vol
+from watchdog.observers import Observer
+from watchdog.events import PatternMatchingEventHandler
+
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+#DEPENDENCIES = ['watchdog']
+_LOGGER = logging.getLogger(__name__)
+
+CONF_PATH = 'folder'
+DOMAIN = "watchdog_file_watcher"
+EVENT_TYPE = "event_type"
+SRC_PATH = "src_path"
+PATTERNS = ["*.txt", "*.py", "*.md", "*.jpg", "*.png"]
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_PATH): cv.isdir})
```

Comment:
This only allows 1 path to be specified. I think that we should take in a list of paths. `vol.All(cv.ensure_list, [cv.isdir])`

Reply:
Agreed. I propose to take a list of paths, with recursive and patterns as optional on each path.